src/main/resources/templates/header.html

```diff
@@ -13,7 +13,7 @@ <h1 class="text-2xl font-bold text-white">
       </h1>
       <div>
           <form action="/cart" method="GET">
-              <input name="cartId" type="hidden">
+              <input name="userId" type="hidden">
```

Comment:
Will we ever use this hidden input?

Reply:
As to my understanding, once we implement a user controller for the user interface, it will return the created userId to the header and index template which I will then use thymeleaf to extract it for cart. Will remove the cookies once that feautre is added

Reply:
I just have this here in preperation for the new feature

Reply:
How would the site get a user ID the first time it is loaded? Do we want to redirect to a type of login screen if the user ID isn't specified?

Reply:
Yeah that can work, or just disable the cart and addtocart button until user logs into website. This is more of @phucla0122 design choice but both work.

Reply:
Yeah I suppose that's out of the scope of this PR.